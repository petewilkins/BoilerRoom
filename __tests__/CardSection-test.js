import 'react-native';
import React from 'react';
import CardSection from '../src/components/CardSection';

import renderer from 'react-test-renderer';

test('renders the CardSection component correctly', () => {
  const tree = renderer.create(
    <CardSection />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
