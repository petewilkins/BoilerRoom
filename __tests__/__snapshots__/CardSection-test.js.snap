// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the CardSection component correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "borderBottomWidth": 1,
      "borderColor": "#DDDDDD",
      "flexDirection": "row",
      "justifyContent": "flex-start",
      "padding": 5,
      "position": "relative",
    }
  }
/>
`;
